edgehog-device-runtime-containers/src/properties/volume.rs

```diff
@@ -23,25 +23,26 @@ use uuid::Uuid;
 
 use crate::service::Id;
 
-use super::{AvailableProp, Client};
+use super::AvailableProp;
 
 const INTERFACE: &str = "io.edgehog.devicemanager.apps.AvailableVolumes";
 
 /// Available volume property.
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub(crate) struct AvailableVolumes<'a> {
     id: &'a Id,
-    created: bool,
 }
 
 impl<'a> AvailableVolumes<'a> {
-    pub(crate) fn new(id: &'a Id, created: bool) -> Self {
-        Self { id, created }
+    pub(crate) fn new(id: &'a Id) -> Self {
+        Self { id }
     }
 }
 
 #[async_trait]
 impl AvailableProp for AvailableVolumes<'_> {
+    type Data = bool;
+
```

Comment:
Is it still valid?

Reply:
Yes, I changed the AvailableProp trait to support the unset. Now it requires an associated type for the property to send